mailpoet/lib/Config/Shortcodes.php

```diff
@@ -215,7 +209,7 @@ public function renderArchiveDate(NewsletterEntity $newsletter) {
     );
   }
 
-  public function renderArchiveSubject(NewsletterEntity $newsletter, ?SubscriberEntity $subscriber, ?SendingQueueEntity $queue) {
+  public function renderArchiveSubject(NewsletterEntity $newsletter, ?SubscriberEntity $subscriber, SendingQueueEntity $queue) {
```

Comment:
This is in contrast to this [former comment](https://github.com/mailpoet/mailpoet/pull/4231#discussion_r920915464), na? but I agree it can be empty/null likes it's always has been like that, PHPStan doesn't catch this because the nullable branch goes through hooks